keps/sig-scheduling/3902-decoupled-taint-manager/README.md

```diff
@@ -173,14 +178,18 @@ Since there are no new API changes, we can skip Alpha and go to Beta directly.
 
 #### GA
 
-* Fix all reported bugs if any.
```

Comment:
From beta requirements: `Performance and scalability tests to verify there is non-negligible performance degradation in node taint-based eviction.`

Was it done? Can you link some evidence?

Reply:
IMO, whatever feature-gate is enabled or not, the feature is implemented by the same code. Nodelifecycle controller starts a goroutine to run `tainteviction.Run` in the `Run` method.
```go
	if !utilfeature.DefaultFeatureGate.Enabled(features.SeparateTaintEvictionController) {
		logger.Info("Starting", "controller", taintEvictionController)
		go nc.taintManager.Run(ctx)
	}
```

What should we do with `Performance and scalability tests`? I have no idea about it. Can you give me some advice? Thanks a lot. 

/cc @Huang-Wei @atosatto @alculquicondor @yuanchen8911 
(reviewers of this PR https://github.com/kubernetes/kubernetes/pull/119208)

Is it covered by existing tests, integration tests, or unit tests? If not, how should we prove it?

- `test/e2e/node/taints.go`: [k8s-triage](https://storage.googleapis.com/k8s-triage/index.html?sig=scheduling&test=NoExecuteTaintManager) no error found.

Reply:
Performance was a graduation criteria for beta. If there were no performance tests - from pure bureaucracy perspective this KEP is not suppose to be in beta now.

Let's retroactively do something about it and if the consensus is that perf tests are not needed, it needs to be explained in a KEP.